analysis/analysis.py

```diff
@@ -1047,5 +1122,5 @@ def test_word_time_relations(self):
 
 
 if __name__ == '__main__':
-    run_analysis()
+    print(run_analysis())
```

Comment:
whoa -- don't change this! everyone else has written their code expecting run_analysis() to just be called in the main function.